LibreNMS/Util/StringHelpers.php

```diff
@@ -48,7 +48,7 @@ public static function niceCase($string)
         $replacements = [
             'bind' => 'BIND',
             'dbm' => 'dBm',
-            'dhcp-stats' => 'DHCP Stats',
+            'dhcp-stats' => 'ISC DHCP Server',
```

Comment:
Other servers could report DHCP stats using the same metrics... seems odd to name it to one server

Reply:
rolled back